craig-rolleston/week5/mon/magic-8ball/app/controllers/rps_controller.rb

```diff
@@ -0,0 +1,16 @@
+
+class RpsController < ApplicationController
+
+    def rps
+    end # secretnumber
+
+    def form
+    end #form
+
+    def result
+
+    end# results
+
+
```

Comment:
you would randomise choice the same way you randomised the magic 8 ball answers and save it in variable. then compare the randomised choice and the user choice. and return who wins. you could do this a number of ways, but you can hard code each situation and return the winner